addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -15,15 +15,30 @@ This binding supports only one thing: The Onkyo AV Receiver.  All supported Onky
 
 This binding can discover the supported Onkyo AV Receivers. At the moment only the following models are supported:
 
+* TX-NR414
+* TX-NR509
+* TX-NR515
+* TX-NR525
 * TX-NR535
+* TX-NR555
+* TX-NR535
+* TX-NR616
 * TX-NR626
+* TX-NR636
 * TX-NR646
+* TX-NR656
+* TX-NR717
+* TX-NR727
+* TX-NR747
+* TX-NR818
+* TX-NR828
+* TX-NR838
 
 ## Binding Configuration
 
 The binding can auto-discover the Onkyo AVRs present on your local network. The auto-discovery is enabled by default. To disable it, you can create a file in the services directory called onkyo.cfg with the following content:
 
-`org.openhab.onkyo:enableAutoDiscovery=false`
+```org.openhab.onkyo:enableAutoDiscovery=false```
```

Comment:
It seems that they all work fine, at least on GitHub.

`onkyo:TX-NR818:myOnkyo [ipAddress="192.168.1.100", port="60128"]`
```onkyo:onkyoAVR:myOnkyo [ipAddress="192.168.1.100", port="60128"]```
```
onkyo:onkyoAVR:myOnkyo [ipAddress="192.168.1.100", port="60128", refreshInterval=30]
```

GitHub recommend:
You can create fenced code blocks by placing triple backticks ``` before and after the code block. **We recommend placing a blank line before and after code blocks to make the raw formatting easier to read.**

https://help.github.com/articles/creating-and-highlighting-code-blocks/

So I will add new lines.